data/tutorials/lg_03_modules.md

```diff
@@ -41,6 +41,10 @@ We can compile the files in one command:
 $ ocamlopt -o hello amodule.ml bmodule.ml
 ```
 
+Note: The source files need to be ordered with the dependencies coming before
+the dependent. In the example above, putting `bmodule.ml` before `amodule.ml`
+will result in an `Unbound module` error.
```

Comment:
```suggestion
Note: It's necessary to place the source files in the correct order. The dependencies must come before
the dependent. In the example above, putting `bmodule.ml` before `amodule.ml`
will result in an `Unbound module` error.
```

Reply:
The original sentence works, if you prefer to keep it. My suggestion breaks it up into two sentences to emphasize the importance of ordering in general and then the necessary order of dependencies/dependent.

Reply:
I like your suggestion of breaking the sentence up in two 🙂 I applied the changes and merged!